deliver/lib/deliver/commands_generator.rb

```diff
@@ -14,6 +14,20 @@ def self.start
       FastlaneCore::UpdateChecker.show_update_status('deliver', Deliver::VERSION)
     end
 
+    def deliverfile_options(skip_verification = false)
+      available_options = Deliver::Options.available_options
+      return available_options unless skip_verification
+
+      # These don't matter for downloading metadata, so verification can be skipped
+      irrelevant_options_keys = [:ipa, :pkg, :app_rating_config_path]
+
+      available_options.each do |opt|
+        next unless irrelevant_options_keys.include?(opt.key)
+        opt.verify_block = nil
+        opt.conflicting_options = nil
+      end
```

Comment:
Let's do an explicit return of `deliverfile_options` to make it more clear that the return value is used.